core-concepts/plugins.md

```diff
@@ -1,200 +1,201 @@
-- Apply **only** to specific stores
-
-Plugins are added to the pinia instance with `pinia.use()`. The simplest example is adding a static property to all stores by returning an object:
+Los plugins son añadidos a la instancia de pinia con `pinia.use()`. El ejemplo más sencillo es añadir una propiedad estática a todos los almacenes mediante el retorno de un objeto:
 
 ```js
 import { createPinia } from 'pinia'
 
-// add a property named `secret` to every store that is created after this plugin is installed
-// this could be in a different file
+// añadir una propiedad llamada `secret` a todos los almacenes creados después 
+// de que este plugin este instalado. Esto puede estar en otro archivo 
+// diferente
 function SecretPiniaPlugin() {
   return { secret: 'the cake is a lie' }
 }
 
 const pinia = createPinia()
-// give the plugin to pinia
+// le damos el plugin a pinia
 pinia.use(SecretPiniaPlugin)
 
-// in another file
+// en otro archivo
 const store = useStore()
 store.secret // 'the cake is a lie'
 ```
 
-This is useful to add global objects like the router, modal, or toast managers.
+Esto es útil para añadir objetos globales como router, modal o toast managers.
 
-## Introduction
+## Introducción {#introduction}
 
-A Pinia plugin is a function that optionally returns properties to be added to a store. It takes one optional argument, a _context_:
+Un plugin de Pinia es una función que opcionalmente retorna propiedades para ser añadidas a un almacén. Toma un argumento opcional, un _contexto_:
 
 ```js
 export function myPiniaPlugin(context) {
-  context.pinia // the pinia created with `createPinia()`
-  context.app // the current app created with `createApp()` (Vue 3 only)
-  context.store // the store the plugin is augmenting
-  context.options // the options object defining the store passed to `defineStore()`
+  context.pinia // el almacén creado con `createPinia()`
+  context.app // la aplicación actual creada con `createApp()` (solo Vue 3)
+  context.store // el almacén aumentado por el plugin
+  context.options // el objeto de opciones que define al almacén pasado por `defineStore()`
   // ...
 }
 ```
 
-This function is then passed to `pinia` with `pinia.use()`:
+Esta función es pasada a `pinia` con `pinia.use()`:
 
 ```js
 pinia.use(myPiniaPlugin)
 ```
 
-Plugins are only applied to stores created **after the plugins themselves, and after `pinia` is passed to the app**, otherwise they won't be applied.
+Los plugins son solo aplicadas a almacenes creados **después de los propios plugins, y después de que `pinia` sea pasada a la aplicación**, de lo contrario no serán aplicados.
 
-## Augmenting a Store
+## Aumentar un Almacén {#augmenting-a-store}
 
-You can add properties to every store by simply returning an object of them in a plugin:
+Puedes añadir propiedades a cualquier almacén solo retornando un objecto de estos mismo en un plugin:
```

Comment:
A. "objeto" instance of "objecto"
B. "esto" instance of "estos" (Change to singular)

Reply:
About B, I believe it could be a better idea use "estas" since it is refering to the properties. Can you see my point? Like:
> Puedes añadir propiedades a cualquier almacén solo retornando un objeto de estas mismas en un plugin

Reply:
got it